IBPSA/ThermalZones/ISO13790/Data/BESTEST600.mo

```diff
@@ -0,0 +1,17 @@
+within IBPSA.ThermalZones.ISO13790.Data;
```

Comment:
The thing is that I think it is useful to add this record as it directly connect the wording "light", "medium", heavy" to a value of heat capacity. Such values are typically not known by users, so it would be difficult for a user to type a value By using a record, users can simply select one option from the menu.  
Regarding the other parameters, it is typically easier to know/calculate them.